src/scss/components/links/_links.scss

```diff
@@ -0,0 +1,59 @@
+.su-link,
+.su-link--action,
+.su-link--external {
+  color: $su-color-digital-red;
+  font-weight: 600;
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $su-color-black;
+  }
+
+  &::after {
+    background-color: $su-color-digital-red;
+  }
+
+  &.mailto {
+    svg {
+      fill: $su-color-digital-red;
+    }
+  }
+}
+
+.su-button,
+.su-button--big {
+  color: $su-color-white;
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $su-color-white;
+  }
+}
+
+.su-button--secondary {
```

Comment:
Isn't this also the default state?

Reply:
Yes and we need to cover the bases for external link icon. Here is examples with and without that code. 
<img width="1023" height="496" alt="Screenshot 2025-08-12 at 7 37 34 AM" src="https://github.com/user-attachments/assets/5ef3db8a-5ae8-4f73-be2f-b778ddc07e21" />
<img width="1020" height="531" alt="Screenshot 2025-08-12 at 7 35 39 AM" src="https://github.com/user-attachments/assets/77ce9ab3-7cc1-455d-a1fe-9d9eb1ac6cf2" />